bittensor/_cli/cli_impl.py

```diff
@@ -379,8 +379,7 @@ def overview(self):
         with console.status(":satellite: Syncing with chain: [white]{}[/white] ...".format(self.config.subtensor.network)):
             for wallet in tqdm(all_hotkeys):
                 nn = subtensor.neuron_for_pubkey( wallet.hotkey.ss58_address )
-                if not nn.is_null:
-                    neurons.append( nn )
+                neurons.append( nn )
```

Comment:
This line is needed to keep to remove de-registered neurons from the sync? What is the justification for removing it?